app/assets/components/TraceInfo.coffee

```diff
@@ -28,13 +28,12 @@ export class Field extends Component
   render: ->
     { title, value, href } = this.props
 
-    if value?
-      value = value.toString()
-
     $ 'div',
       $ 'h4', title
       do =>
-        if value
-          $ 'a', title: value, href: href, value
+        if value? && href
```

Comment:
This isn't quite correct yet, as some of the other fields omit the `href` prop as well. What I want to really check is whether `value` is itself a VDOM node, but I didn't know how...